docs/developer/plugin-list.asciidoc

```diff
@@ -216,14 +216,27 @@ which also contains the timelion APIs and backend, look at the vis_type_timelion
 
 
 |<<uiactions-plugin>>
-|An API for:
-
-- creating custom functionality (`actions`)
-- creating custom user interaction events (`triggers`)
-- attaching and detaching `actions` to `triggers`.
-- emitting `trigger` events
-- executing `actions` attached to a given `trigger`.
-- exposing a context menu for the user to choose the appropriate action when there are multiple actions attached to a single trigger.
+|UI Actions plugins provides API to manage *triggers* and *actions*.
+
+*Trigger* is an abstract description of user's intent to perform an action
+(like user clicking on a value inside chart). It allows us to do runtime
+binding between code from different plugins. For, example one such
+trigger is when somebody applies filters on dashboard; another one is when
```

Comment:
the trigger is actually when someone clicks something inside a chart isn;t it ?

Reply:
I was writing it from a standpoint that it does not necessarily need to be a "click" it could be triggered by anything. Also, even the "click" could be actually a "touch" or it could be executed using a11y.